pom.xml

```diff
@@ -14,11 +14,11 @@
     <packaging>atlassian-plugin</packaging>
 
     <properties>
-        <bitbucket.version>5.5.0</bitbucket.version>
+        <bitbucket.version>6.0.1</bitbucket.version>
```

Comment:
Will users of BitBucket 6.0.0 be able to use the plugin?

Reply:
It works for me on 6+.